precompile/contracts/test_utils/test_precompile.go

```diff
@@ -0,0 +1,84 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package test_utils
```

Comment:
can we rename this to `testutils` without underscore?

Reply:
also moved it out of contracts